meta/runtime.yml

```diff
@@ -189,6 +190,13 @@ plugin_routing:
         warning_text: >-
           rds_param_group has been renamed to rds_instance_param_group.
           Please update your tasks.
+    ec2_vpc_peer:
+      redirect: amazon.aws.ec2_vpc_peer
+      deprecation:
+        removal_version: 11.0.0
```

Comment:
Is 1 year long enough?

I would be inclined to push this out to 13, and start the deprecation of aliases from the mass-rename...

Reply:
Updated.